guides/10_Video_Processor.md

```diff
@@ -188,42 +206,70 @@ class VideoLoadImageProcessor implements VideoFrameProcessor {
 }
 ```
 
-## Video Processing Usage
-### Custom processor usage during meeting
+## Additional Video Processing Use-Cases
+
+### Custom processor usage during meeting preview
+
+Local video post processing can be previewed before transmitting to remote clients just for a normal device.
 
 ```typescript
 import {
   DefaultVideoTransformDevice
 } from 'amazon-chime-sdk-js';
 
 const stages = [new VideoResizeProcessor(4/3)]; // constructs  processor
-
+const videoElement = document.getElementById('video-preview') as HTMLVideoElement;
 const transformDevice = new DefaultVideoTransformDevice(
   logger,
   'foobar', // device id string
   stages
 );
 
 await meetingSession.audioVideo.startVideoInput(transformDevice);
-meetingSession.audioVideo.startLocalVideo();
-
+meetingSession.audioVideo.startVideoPreviewForVideoInput(videoElement);
 ```
 
-### Custom processor usage during meeting preview
+### Custom video processor usage for content share
+
+The API `ContentShareControllerFacade.startContentShare` does not currently support passing in a `VideoTransformDevice` or similar. But the `DefaultVideoTransformDevice` makes it straight forward to apply transforms on a given `MediaStream`, and output a new `MediaStream`.
+
+Note that for screen share usage we need to use `ContentShareMediaStreamBroker` directly rather then the helper function `ContentShareControllerFacade.startContentShareFromScreenCapture`.
 
 ```typescript
 import {
   DefaultVideoTransformDevice
 } from 'amazon-chime-sdk-js';
 
-const processor = new VideoResizeProcessor(4/3);
-const videoElement = document.getElementById('video-preview') as HTMLVideoElement;
+const framerate = 30;
+var mediaStream: MediaStream = new ContentShareMediaStreamBroker(logger).acquireScreenCaptureDisplayInputStream(
```

Comment:
Should we just tell them to use the browser API `GetDisplayMedia` ?

Reply:
I originally thought to abstract away the creation of the media capture constraints but maybe its not a big deal (don't need the `getUserMedia` fork. I will switch to `getDisplayMedia` as that avoids having builder need to know what a `MediaStreamBroker` is :P.